app/assets/javascripts/discourse/app/lib/register-topic-footer-button.js

```diff
@@ -97,6 +96,7 @@ export function getTopicFooterButtons() {
           const discourseComputedButon = {};
 
           discourseComputedButon.id = button.id;
+          discourseComputedButon.type = button.type;
```

Comment:
Is there a reason we spell this as `Buton`?

Reply:
no :p